exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/PrometheusMetricNameMapperTest.java

```diff
@@ -1,163 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package io.opentelemetry.exporter.prometheus;
-
-import static io.opentelemetry.exporter.prometheus.TestConstants.DELTA_HISTOGRAM;
-import static io.opentelemetry.exporter.prometheus.TestConstants.DOUBLE_GAUGE;
-import static io.opentelemetry.exporter.prometheus.TestConstants.MONOTONIC_CUMULATIVE_LONG_SUM;
-import static io.opentelemetry.exporter.prometheus.TestConstants.SUMMARY;
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-import io.opentelemetry.sdk.metrics.data.MetricData;
-import io.opentelemetry.sdk.metrics.internal.data.ImmutableMetricData;
-import java.util.concurrent.atomic.AtomicInteger;
-import java.util.function.BiFunction;
-import java.util.stream.Stream;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.Arguments;
-import org.junit.jupiter.params.provider.MethodSource;
-
-class PrometheusMetricNameMapperTest {
-
-  @Test
-  void prometheusMetricNameMapperCaching() {
-    AtomicInteger count = new AtomicInteger();
-    BiFunction<MetricData, PrometheusType, String> delegate =
-        (metricData, prometheusType) ->
-            String.join(
-                "_",
-                metricData.getName(),
-                prometheusType.name(),
-                Integer.toString(count.incrementAndGet()));
-    PrometheusMetricNameMapper mapper = new PrometheusMetricNameMapper(delegate);
-
-    assertThat(mapper.apply(MONOTONIC_CUMULATIVE_LONG_SUM, PrometheusType.GAUGE))
-        .isEqualTo("monotonic.cumulative.long.sum_GAUGE_1");
-    assertThat(mapper.apply(MONOTONIC_CUMULATIVE_LONG_SUM, PrometheusType.GAUGE))
-        .isEqualTo("monotonic.cumulative.long.sum_GAUGE_1");
-    assertThat(mapper.apply(MONOTONIC_CUMULATIVE_LONG_SUM, PrometheusType.GAUGE))
-        .isEqualTo("monotonic.cumulative.long.sum_GAUGE_1");
-    assertThat(mapper.apply(MONOTONIC_CUMULATIVE_LONG_SUM, PrometheusType.GAUGE))
-        .isEqualTo("monotonic.cumulative.long.sum_GAUGE_1");
-    assertThat(mapper.apply(MONOTONIC_CUMULATIVE_LONG_SUM, PrometheusType.GAUGE))
-        .isEqualTo("monotonic.cumulative.long.sum_GAUGE_1");
-    assertThat(count).hasValue(1);
-  }
-
-  @ParameterizedTest
-  @MethodSource("provideRawMetricDataForTest")
-  void metricNameSerializationTest(MetricData metricData, String expectedSerializedName) {
-    assertEquals(
-        expectedSerializedName,
-        PrometheusMetricNameMapper.INSTANCE.apply(
-            metricData, PrometheusType.forMetric(metricData)));
-  }
-
-  private static Stream<Arguments> provideRawMetricDataForTest() {
```

Comment:
IIUC, these test cases are still valid. If this is indeed the case, is there a way we could retain these test cases ?

Reply:
Check out #6138 where I restore these test cases.